sdk/keyvault/Azure.Security.KeyVault.Keys/README.md

```diff
@@ -182,7 +185,7 @@ Console.WriteLine(key.DeletedDate);
 This example lists all the keys in the specified Key Vault.
 
 ```c#
-IEnumerable<Response<KeyBase>> allKeys = client.GetKeys();
+IEnumerable<Azure.Response<KeyBase>> allKeys = client.GetKeys();
```

Comment:
I'm not a huge fan of fully qualified namespaces in the sample code. I wonder if there's some way to document the using statements needed to not fully qualify

Reply:
Agreed, my mistake. We can just put

```c#
using Azure;
```

above the snippet I think?